packages/postcss-normalize-url/src/__tests__/index.js

```diff
@@ -271,7 +271,7 @@ test(
   'should handle protocol relative urls',
   processCSS(
     'h1{background:url(//website.com:80/image.png)}',
-    'h1{background:url(//website.com/image.png)}'
+    'h1{background:url(//website.com:80/image.png)}'
```

Comment:
We should not remove `:80` because it is unnecessary

Reply:
> We should not remove `:80` because it is unnecessary

Currently it's not removing !!! you want it to be removed ?

Reply:
Yes, it should be removed

Reply:
but will it be safe ?
like it will work fine for `80` but if its different than HTTP port like `3000`, then the port should be kept as it is right !

Reply:
@evilebottnawi